velox/type/tz/TimeZoneMap.cpp

```diff
@@ -125,6 +126,51 @@ const TTimeZoneIndex& getTimeZoneIndex() {
   return timeZoneIndex;
 }
 
+TJavaShortIdIndex buildJavaShortIdIndex() {
+  static TJavaShortIdIndex javaShortIdMap = {
+      {"EST", "-05:00"},
+      {"HST", "-10:00"},
+      {"MST", "-07:00"},
+      {"ACT", "Australia/Darwin"},
+      {"AET", "Australia/Sydney"},
+      {"AGT", "America/Argentina/Buenos_Aires"},
+      {"ART", "Africa/Cairo"},
+      {"AST", "America/Anchorage"},
+      {"BET", "America/Sao_Paulo"},
+      {"BST", "Asia/Dhaka"},
+      {"CAT", "Africa/Harare"},
+      {"CNT", "America/St_Johns"},
+      {"CST", "America/Chicago"},
+      {"CTT", "Asia/Shanghai"},
+      {"EAT", "Africa/Addis_Ababa"},
+      {"ECT", "Europe/Paris"},
+      {"IET", "America/Indiana/Indianapolis"},
+      {"IST", "Asia/Kolkata"},
+      {"JST", "Asia/Tokyo"},
+      {"MIT", "Pacific/Apia"},
+      {"NET", "Asia/Yerevan"},
+      {"NST", "Pacific/Auckland"},
+      {"PLT", "Asia/Karachi"},
+      {"PNT", "America/Phoenix"},
+      {"PRT", "America/Puerto_Rico"},
+      {"PST", "America/Los_Angeles"},
+      {"SST", "Pacific/Guadalcanal"},
+      {"VST", "Asia/Ho_Chi_Minh"},
+  };
+  TJavaShortIdIndex lower;
+  lower.reserve(javaShortIdMap.size());
+  for (auto& it : javaShortIdMap) {
+    lower[boost::algorithm::to_lower_copy(it.first)] =
```

Comment:
There is a 'sanitizeName' method to convert name as lower case.

Reply:
Including `FunctionSignature.h` here seems a bit odd; I think it's better to stick with Boost to maintain consistency with the handling logic of `TimeZoneIndex`.